code/game/gamemodes/game_mode.dm

```diff
@@ -362,6 +362,28 @@
 					player.mind.token_picked = TRUE
 					return player.mind
 
+	
+	for(var/datum/mind/mind in candidates)
+		var/antagrounds = 0
+		var/antagckey = mind.key
+		if(!SSdbcore.Connect())
+			log_world("Failed to connect to database in load_antaground().")
+			WRITE_FILE(GLOB.world_game_log, "Failed to connect to database in load_antaground().")
+			continue
+
+		var/datum/DBQuery/query_load_antagrounds = SSdbcore.NewQuery("SELECT antagrounds FROM [format_table_name("player")] WHERE ckey = :antagckey", list("antagckey" = antagckey))
```

Comment:
A refactor for a future PR perhaps, but maybe we should be loading antag weight, antag tokens, antag rounds, etc **once** with a single query when the player connects to the server and cache it instead of running a whole bunch of queries as the round starts.